.github/workflows/ccpp-tests.yml

```diff
@@ -112,12 +112,13 @@ jobs:
       with:
         name: docs
         path: mybuild_docs/sphinx/html
-    - name: Link Checker
-      uses: peter-evans/link-checker@v1
+    - name: (lychee) Link Checker
+      uses: lycheeverse/lychee-action@v1
```

Comment:
https://github.com/lycheeverse/lychee-action
use latest version lycheeverse/lychee-action@v1.5.4

Reply:
https://github.com/lycheeverse/lychee-action

"If you always want to use the latest features but avoid breaking changes, you can replace the version with
lycheeverse/lychee-action@v1"